jerry-core/include/jerryscript-debugger.h

```diff
@@ -28,12 +28,22 @@ extern "C"
  */
 
 /**
+ * Types for the client source wait and run method.
+ */
+typedef enum
+{
+  JERRY_DEBUGGER_WAIT_AND_RUN_FAIL = 0, /**< error occured while receive */
+  JERRY_DEBUGGER_WAIT_AND_RUN_SUCCESS = 1, /**< the source has been received */
```

Comment:
I would prefer JERRY_DEBUGGER_WAIT_AND_RUN_RECEIVED to specify that the source is arrived. It does not mean the source execution is successful.